src/main/java/emissary/core/IBaseDataObjectXmlCodecs.java

```diff
@@ -826,4 +831,55 @@ public static Method getIbdoMethod(final String methodName, final Class<?>... pa
             throws NoSuchMethodException {
         return IBaseDataObject.class.getMethod(methodName, parameterTypes);
     }
+
+    public static boolean requiresEncoding(final byte[] utf8Bytes) {
```

Comment:
Recommend removing `requiresEncoding` method and updating any calls to `requiresEncoding` to call ByteUtil#containsNonIndexableBytes instead.